### DISCUSSION TITLE
Working On an Issue

### DISCUSSION THREAD

**Discussion by Emi98nat** [Emi98nat/restaurant_app#7] [General]
Hey, Peers gather here let's solve the riddle of working on issues

**Comment by Emi98nat** [comment]
@Vicksbabe58 great to have you on board

**Reply by Vicksbabe58** [reply]
Thanks, happy to be here

**Comment by Emi98nat** [comment]
@Vicksbabe58 have you pulled the repo locally

**Reply by Vicksbabe58** [reply]
Yes, I have , am using feature-menu-5 branch

**Reply by Emi98nat** [reply]
@Vicksbabe58 great for now just use the images on figma, we will all discuss and propose other great images we'll get from our images tools

**Reply by Emi98nat** [reply]
I am adding more tools for the CSS section I need to get to the project and make some new commits on the landing Page

**Reply by Vicksbabe58** [reply]
@Emi98nat  alright, you said something about working on an issue ... what is the issue
